Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: image
+
+IMAGE_NAME ?= codeclimate/codeclimate-gofmt
+
+update:
+	docker run \
+    --rm --interactive \
```

Comment:
Indentation is off on these lines: should be a hard tab followed by two soft spaces so it's indented a bit further than `docker`.